torch_xla/csrc/tensor_ops.cpp

```diff
@@ -59,15 +59,22 @@ XLATensor Cross(const XLATensor& input, const XLATensor& other,
 }
 
 XLATensor KlDivBackward(const XLATensor& grad_output, const XLATensor& input,
-                        const XLATensor& target, ReductionMode reduction) {
+                        const XLATensor& target, ReductionMode reduction,
+                        bool log_target) {
```

Comment:
This the backward formula for `input`. What about the backward for `target`?

Reply:
See
`tools/autograd/derivatives.yaml` and
`tools/autograd/templates/Functions.cpp`.
Check my update as well, I had to modify it because of wrong autograd usage...

Reply:
I chat with @ailzhang , `kl_div_target_backward` is general for all backends. We only need to overwrite `kl_div_backward` because it is device specific. Note that there is a `kl_div_backward_cpu` and `kl_div_backward_kuda`

Reply:
OK, I did not know ;)

Reply:
Thanks for reviewing, I only know that couple minutes earlier than you 😉